fronts-client/src/actions/Cards.ts

```diff
@@ -322,6 +323,57 @@ export const mayResetImmersive = ({
 	}
 };
 
+/**
+ * If you move a card with a 4:5 replacement video to a 5:4 slot (or vice versa),
+ * we revert to the trail image and disable the replacement video
+ * */
+export const mayResetVideoReplace = ({
+	from,
+	to,
+	card,
+	persistTo,
+	state,
+}: UpdateCardMetaParams) => {
+	if (
+		to.type === 'group' &&
+		persistTo === 'collection' &&
+		from?.id !== to.id &&
+		card.meta?.videoReplace
+	) {
+		const videoAtom: Atom = card.meta.replacementVideoAtom as Atom;
+		// TODO: video atom doesn't have any direct info on aspect ratio
+		//  we can infer something from the posterImage (if available)
+		//  Ideally we would add some data to the atom to make this easier
+		const dimensions =
+			videoAtom && videoAtom.atomType
```

Comment:
Can we shorthand this to 
```suggestion
			videoAtom?.atomType
```